lib/AutoPodArgsPass.cpp

```diff
@@ -49,6 +49,14 @@ class AutoPodArgsPass : public ModulePass {
 
   // Makes kernel |F| use |impl| as the pod arg implementation.
   void AddMetadata(Function &F, clspv::PodArgImpl impl);
+
+  // Returns true if |type| contains an array. Does not consider pointers since
```

Comment:
Nit:  Consider "does not look through pointers" instead of "does not consider pointers"  ?